docs/3_development/1_development-flow/2_development.md

```diff
@@ -41,4 +41,8 @@ Our repositories typically have a template for pull requests, therefore writing
 - Add the according labels based on the project
 - Ensure you adhere to the repositories PR template.
     - If a template is not provided contact you Tech Lead or Project Manager for insights.
-- Always ensure you are descriptive, when explaining the nature of your pull request.
\ No newline at end of file
+- Always ensure you are descriptive, when explaining the nature of your pull request.
+
+## 5. Approaches in use
+
+In many of the [ChainSafe Products](https://handbook.chainsafe.dev/departments/product) development cycles OneFlow approach is used. You can find more [detailed description here](https://handbook.chainsafe.dev/development/pages/oneflow).
```

Comment:
```suggestion
In many of the [ChainSafe Products](https://handbook.chainsafe.dev/departments/product) development cycles OneFlow approach is used. You can find more [detailed description here](/pages/oneflow).
```

Should be relative url

Reply:
@kalambet Might be better to link like this: https://github.com/ChainSafe/engineering-handbook/blob/main/docs/3_development/4_documentation.md?plain=1#L46
so that it opens in new page as it's hard to navigate back